Controller/Component/Auth/ApiAuthenticate.php

```diff
@@ -12,7 +12,7 @@ class ApiAuthenticate extends BaseAuthenticate {
 	/**
 	 * @var string
 	 */
-	protected $_authHeaderName = 'authorization';
+	protected $_authHeaderName = 'Authorization';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "$_authHeaderName" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)